spec/gs1/barcode/healthcare_spec.rb

```diff
@@ -27,6 +27,17 @@
         expect(subject.serial_number).to eq(GS1::SerialNumber.new('123123'))
       end
     end
+
+    context "01070462606504112101582353425597816838\u001E1721093010184238131" do
```

Comment:
Perhaps a name that is more descriptive of the case being tested?

Reply:
Output:
![image](https://user-images.githubusercontent.com/1053201/53006375-642b9680-3435-11e9-9ebc-a363bbebeb5c.png)

How would you name this?

Reply:
Result:
![image](https://user-images.githubusercontent.com/1053201/53006741-1c593f00-3436-11e9-9e45-15f77b9829d2.png)